keyboards/millipad/readme.md

```diff
@@ -0,0 +1,28 @@
+# millipad
+
+![millipad](https://i.imgur.com/ZM2qel5.jpg)
+
+An open source 2x6 macropad with rotary encoder. Designed to be cheap and simple.
+
+## Bootloader Mode: 
+Press the hardware reset button located to the left of the rotary encoder.
+
+## More Pictures:
+![PCB Layout](https://i.imgur.com/ilhfBfQ.png)
+![Front of PCB](https://i.imgur.com/UiQkmuv.jpg)
+![Back of PCB](https://i.imgur.com/Inh6UTL.jpg)
+
+
+* Keyboard Maintainer: [Jirou](https://github.com/GLozares)
+* Hardware Supported: Jirou.Design Millipad
+* Hardware Availability: Open Source at https://github.com/GLozares/millipad
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?

Reply:
I moved the location of the bootloader instructions to be closer to the make example and made it clearer in my latest commit.